packages/clay-management-toolbar/src/ClayManagementToolbar.js

```diff
@@ -606,6 +607,15 @@ ClayManagementToolbar.STATE = {
 	 */
 	spritemap: Config.string().required(),
 
+	/**
+	 * Flag to indicate that the toolbar supports bulk selection.
+	 * @default false
+	 * @instance
+	 * @memberof ClayManagementToolbar
+	 * @type {boolean}
+	 */
+	supportsBulkActions: Config.bool().value(false),
```

Comment:
Hey @matuzalemsteles, the `showSelectAllButton` logic is a bit more complicated than that... it shouldn't show by default, even if the `bulkAction` feature is enabled, so `false` still sounds like the proper default. You can see the whole definition here https://docs.google.com/document/d/1Aov9rThS1bcVscmS36DyCbpHdrM0OcEJZhA4w4_kHR4/edit?usp=drive_web&ouid=118239731538492698708

Parts of it are implemented here, but as @carloslancha said, we've tried to keep this component dumb so it can be easily controlled from above in Portal.